src/compiler/transformers/ts.ts

```diff
@@ -613,9 +676,9 @@ namespace ts {
          * @param statements A statement list to which to add the declaration.
          * @param node A ClassDeclaration node.
          * @param name The name of the class.
-         * @param hasExtendsClause A value indicating whether the class has an extends clause.
+         * @param facts Precomputed facts about the clas.

```

Comment:
"class". Although the documentation here isn't too useful since that's just a description of the type. (Same for `@param node`)